cypress/integration/profile/create-tls.real.ui.spec.ts

```diff
@@ -0,0 +1,38 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2021
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+// Test the create, read, update and delete profile
+
+describe('Create Read Update Delete Profile Test Suite', () => {
```

Comment:
It looks like we have a create test already as well as a delete test. I would look at those for what to base future tests off of . That said, i think an update test would be useful, but needs to be in its own file and expanded to include api status code checks, mocks, and expectations around all values that change